src/puppetlabs/puppetdb/command.clj

```diff
@@ -69,7 +69,7 @@
             [puppetlabs.puppetdb.utils :as utils]
             [slingshot.slingshot :refer [try+ throw+]]
             [puppetlabs.puppetdb.mq-listener :as mql]
-            [puppetlabs.puppetdb.command.constants :refer [command-names]]
+            [puppetlabs.puppetdb.command.constants :as cmd-consts]
```

Comment:
Actually I might take that back a bit - looks like some of the changes I was thinking about perhaps aren't "additional" because the namespace is changing (because they weren't refers to begin with); so dunno.